src/vaadin-grid-pro-inline-editing-mixin.html

```diff
@@ -248,6 +248,15 @@
       this.__edited = {cell, column, model};
       column._startCellEdit(cell, model);
 
+      this.dispatchEvent(new CustomEvent('cell-edit-started', {
+        detail: {
+          index: model.index,
+          item: model.item,
+          path: column.path
+        },
+        bubbles: true,
```

Comment:
do we need this event to bubble or is it just to be aligned with other events?

Reply:
Removed.